README.md

```diff
@@ -0,0 +1 @@
+![main screen]("/screenshots/Screenshot_1581457330.png")
```

Comment:
You'll want to resize your images, one way I know how is like this:

<img src="/screenshots/Screenshot_1581457330.png"
     width=150;  margin-right= 10px;/>